phpcs.xml.dist

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
```

Comment:
This file might be used from Doctrine/CodingStandard, right? This might be unnecessary duplication

Reply:
Then loading it in CLI with `--config`

Reply:
I'd say we should have a `phpcs.xml.dist` for each project, the same way each project has it's own `phpunit.xml.dist`.

The file on `doctrine/coding-standard` is to validate `doctrine/coding-standard` code.

Reply:
Agreed, each project should have its own dist file.